asciidoctor.gemspec

```diff
@@ -42,7 +42,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'cucumber', '~> 3.1.0'
   # erubis is needed for testing alternate eRuby impls
   s.add_development_dependency 'erubis', '~> 2.7.0'
-  s.add_development_dependency 'haml', '~> 5.0.0'
+  s.add_development_dependency 'haml', '>= 4.0', '< 5.0'
```

Comment:
Can you do this in the Gemfile instead? I still want to test on Haml 5 for other Rubies.

It would look something like this:

```rb
if RUBY_ENGINE == 'truffleruby'
  gem 'haml', '~> 4.0'
end
```

Reply:
done